examples/rules_ios/test/fixtures/bwb.xcodeproj/rules_xcodeproj/bazel/Debug-swift_debug_settings.py

```diff
@@ -18,28 +18,16 @@
 	},
 	"arm64-apple-ios LibDynamic.framework": {
-		"c": "-iquote$(PROJECT_DIR) -iquote$(PROJECT_DIR)/bazel-out/ios-arm64-min15.0-applebin_ios-ios_arm64-dbg-STABLE-2/bin -ivfsoverlay$(CURRENT_EXECUTION_ROOT)/bazel-out/ios-arm64-min15.0-applebin_ios-ios_arm64-dbg-STABLE-2/bin/Lib/Lib_swift_vfs.yaml -F/build_bazel_rules_ios/frameworks -I$(PROJECT_DIR)/bazel-out/ios-arm64-min15.0-applebin_ios-ios_arm64-dbg-STABLE-2/bin/Lib/Lib_private_hmap.hmap -I$(PROJECT_DIR)/bazel-out/ios-arm64-min15.0-applebin_ios-ios_arm64-dbg-STABLE-2/bin/Lib/Lib_public_hmap.hmap -I$(PROJECT_DIR)/bazel-out/ios-arm64-min15.0-applebin_ios-ios_arm64-dbg-STABLE-2/bin/Lib/Lib_private_angled_hmap.hmap -iquote$(PROJECT_DIR)/bazel-out/ios-arm64-min15.0-applebin_ios-ios_arm64-dbg-STABLE-2/bin/Lib/Lib_private_hmap.hmap -D__SWIFTC__ -I$(PROJECT_DIR) -O0 -DDEBUG=1 -fstack-protector -fstack-protector-all",
-		"f": [
-			"/build_bazel_rules_ios/frameworks"
-		]
+		"c": "-iquote$(PROJECT_DIR) -iquote$(PROJECT_DIR)/bazel-out/ios-arm64-min15.0-applebin_ios-ios_arm64-dbg-STABLE-2/bin -ivfsoverlay$(CURRENT_EXECUTION_ROOT)/bazel-out/ios-arm64-min15.0-applebin_ios-ios_arm64-dbg-STABLE-2/bin/Lib/Lib_swift_vfs.yaml -F/build_bazel_rules_ios/frameworks -I$(PROJECT_DIR)/bazel-out/ios-arm64-min15.0-applebin_ios-ios_arm64-dbg-STABLE-2/bin/Lib/Lib_private_hmap.hmap -I$(PROJECT_DIR)/bazel-out/ios-arm64-min15.0-applebin_ios-ios_arm64-dbg-STABLE-2/bin/Lib/Lib_public_hmap.hmap -I$(PROJECT_DIR)/bazel-out/ios-arm64-min15.0-applebin_ios-ios_arm64-dbg-STABLE-2/bin/Lib/Lib_private_angled_hmap.hmap -iquote$(PROJECT_DIR)/bazel-out/ios-arm64-min15.0-applebin_ios-ios_arm64-dbg-STABLE-2/bin/Lib/Lib_private_hmap.hmap -D__SWIFTC__ -I$(PROJECT_DIR) -O0 -DDEBUG=1 -fstack-protector -fstack-protector-all"
 	},
 	"arm64-apple-ios MixedAnswer": {
-		"c": "-iquote$(PROJECT_DIR) -iquote$(PROJECT_DIR)/bazel-out/ios-arm64-min15.0-applebin_ios-ios_arm64-dbg-STABLE-2/bin -ivfsoverlay$(CURRENT_EXECUTION_ROOT)/bazel-out/ios-arm64-min15.0-applebin_ios-ios_arm64-dbg-STABLE-2/bin/iOSApp/Source/CoreUtilsMixed/MixedAnswer/MixedAnswer_swift_vfs.yaml -F/build_bazel_rules_ios/frameworks -I$(PROJECT_DIR)/bazel-out/ios-arm64-min15.0-applebin_ios-ios_arm64-dbg-STABLE-2/bin/iOSApp/Source/CoreUtilsMixed/MixedAnswer/MixedAnswer_private_hmap.hmap -I$(PROJECT_DIR)/bazel-out/ios-arm64-min15.0-applebin_ios-ios_arm64-dbg-STABLE-2/bin/iOSApp/Source/CoreUtilsMixed/MixedAnswer/MixedAnswer_public_hmap.hmap -I$(PROJECT_DIR)/bazel-out/ios-arm64-min15.0-applebin_ios-ios_arm64-dbg-STABLE-2/bin/iOSApp/Source/CoreUtilsMixed/MixedAnswer/MixedAnswer_private_angled_hmap.hmap -iquote$(PROJECT_DIR)/bazel-out/ios-arm64-min15.0-applebin_ios-ios_arm64-dbg-STABLE-2/bin/iOSApp/Source/CoreUtilsMixed/MixedAnswer/MixedAnswer_private_hmap.hmap -D__SWIFTC__ -I$(PROJECT_DIR) -O0 -DDEBUG=1 -fstack-protector -fstack-protector-all",
-		"f": [
-			"/build_bazel_rules_ios/frameworks"
```

Comment:
I think this is actually desired, and shouldn't break anything. Can I get some confirmation that lldb inside rules_ios swift targets still works nicely after this change? @thiagohmcruz @mattrobmattrob @luispadron

Reply:
hmm maybe not? It seems stuck trying to `po`. Im hitting breakpoints just fine and see the resolved values of what im trying to `po` in the navigator in Xcode. 

Let me see if this is the same on `1.4.0`

Reply:
Ok yeah both `1.4.0` and this PR are working with `po` on a swift module. The hang im seeing is when i tried to `po` from our main app target which i imagine is just slow because of its size. 

TL:DR; This change doesn't break `po`

<img width="784" alt="Screenshot 2023-04-18 at 5 44 59 PM" src="https://user-images.githubusercontent.com/13840545/232912112-de7294e3-7d15-49fc-accd-09253fefc9b1.png">

Reply:
@mattrobmattrob This worked for you as well?

Reply:
I didn't test it but I can. Will let you know by lunch time.

Reply:
So with a breakpoint in `rules_ios.ios_unit_test` underlying Swift target, I'm getting:
```
(lldb) po asset
Cannot create Swift scratch context (couldn't create a ClangImporter)error: Could not create an expression for language swift
(lldb) po asset
error: Could not create an expression for language swift
```
And then in a breakpoint on that static `rules_ios.apple_framework` underlying Swift target linked into that `rules_ios.ios_unit_test`:
```
(lldb) po self
Cannot create Swift scratch context (couldn't create a ClangImporter)error: Could not create an expression for language swift
(lldb) po self
error: Could not create an expression for language swift
```

Reply:
Is that new to this PR? And regardless, can you send me an LLDB types log (DM)?